nextflow.config

```diff
@@ -208,73 +172,89 @@ env {
     JULIA_DEPOT_PATH = "/usr/local/share/julia"
 }
 
-// Capture exit codes from upstream processes when piping
-process.shell = ['/bin/bash', '-euo', 'pipefail']
+// Set bash options
+process.shell = """\
+bash
+
+set -e # Exit if a tool returns a non-zero status/exit code
+set -u # Treat unset variables and parameters as an error
+set -o pipefail # Returns the status of the last command to exit with a non-zero status or zero if all successfully execute
+set -C # No clobber - prevent output redirection from overwriting files.
+"""
 
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'seqeralabs/nf-dragen'
-    author          = """Harshil Patel, Graham Wright"""
+    author          = """Adam Talbot, Harshil Patel, Graham Wright""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        [
+            name: 'Adam Talbot',
+            affiliation: 'Seqera',
+            email: 'adam.talbot@seqera.io',
+            github: '@adamrtalbot',
+            contribution: ['maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Harshil Patel',
+            affiliation: 'Seqera',
+            email: 'harshil.patel@seqera.io',
+            github: '@drpatelh',
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Graham Wright',
+            affiliation: 'Seqera',
+            email: 'graham.wright@seqera.io',
+            github: '@gwright99',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+    ]
     homePage        = 'https://github.com/seqeralabs/nf-dragen'
```

Comment:
```suggestion
    homePage        = 'https://github.com/seqeralabs/dragen'
```

Reply:
yeah, I noticed that but you called it just dragen somewhere, so I assumed that was the name, sorry about that